install/kubernetes/cilium/templates/cilium-operator-deployment.yaml

```diff
@@ -105,62 +102,60 @@ spec:
-          value: {{ .Values.global.azure.subscriptionID }}
+          value: {{ .Values.azure.subscriptionID }}
 {{- end }}
-{{- if .Values.global.azure.tenantID }}
+{{- if .Values.azure.tenantID }}
         - name: AZURE_TENANT_ID
-          value: {{ .Values.global.azure.tenantID }}
+          value: {{ .Values.azure.tenantID }}
 {{- end }}
-{{- if .Values.global.azure.resourceGroup }}
+{{- if .Values.azure.resourceGroup }}
         - name: AZURE_RESOURCE_GROUP
-          value: {{ .Values.global.azure.resourceGroup }}
+          value: {{ .Values.azure.resourceGroup }}
 {{- end }}
-{{- if .Values.global.azure.clientID }}
+{{- if .Values.azure.clientID }}
         - name: AZURE_CLIENT_ID
-          value: {{ .Values.global.azure.clientID }}
+          value: {{ .Values.azure.clientID }}
 {{- end }}
-{{- if .Values.global.azure.clientSecret }}
+{{- if .Values.azure.clientSecret }}
         - name: AZURE_CLIENT_SECRET
-          value: {{ .Values.global.azure.clientSecret }}
+          value: {{ .Values.azure.clientSecret }}
 {{- end }}
-{{- if contains "/" .Values.image }}
-        image: "{{ .Values.image }}"
-{{- else }}
-{{- if .Values.global.eni }}
-        image: "{{ .Values.global.registry }}/{{ .Values.image }}-aws:{{ .Values.global.tag }}"
-{{- else if .Values.global.azure.enabled }}
-        image: "{{ .Values.global.registry }}/{{ .Values.image }}-azure:{{ .Values.global.tag }}"
-{{- else if .Values.global.ipam.operator }}
-        image: "{{ .Values.global.registry }}/{{ .Values.image }}-generic:{{ .Values.global.tag }}"
-{{- else }}
-        image: "{{ .Values.global.registry }}/{{ .Values.image }}:{{ .Values.global.tag }}"
+{{- range $key, $value := .Values.operator.extraEnv }}
+        — name: {{ $key }}
+          value: {{ $value }}
 {{- end }}
+{{- if .Values.eni }}
+        image: {{ .Values.operator.image.repository }}-aws:{{ .Values.operator.image.tag }}
+{{- else if .Values.azure.enabled }}
+        image: {{ .Values.operator.image.repository }}-azure:{{ .Values.operator.image.tag }}
+{{- else }}
+        image: {{ .Values.operator.image.repository }}-generic:{{ .Values.operator.image.tag }}
 {{- end }}
-        imagePullPolicy: {{ .Values.global.pullPolicy }}
+        imagePullPolicy: {{ .Values.operator.image.pullPolicy }}
         name: cilium-operator
-{{- if .Values.global.prometheus.enabled }}
+{{- if .Values.operator.prometheus.enabled }}
         ports:
-        - containerPort: {{ .Values.global.operatorPrometheus.port }}
-          hostPort: {{ .Values.global.operatorPrometheus.port }}
+        - containerPort: {{ .Values.operator.prometheus.port }}
+          hostPort: {{ .Values.operator.prometheus.port }}
           name: prometheus
           protocol: TCP
 {{- end }}
         livenessProbe:
           httpGet:
-{{- if .Values.global.ipv4.enabled }}
+{{- if .Values.ipv4.enabled }}
             host: '127.0.0.1'
 {{- else }}
-            host: '::1'
+            host: '[::1]'
```

Comment:
why this change? as far as I remember kubelet will take this literally and then will re-add another group of `[``]`

Reply:
I don't recall making this change, but can revert it.

Reply:
addressed all the above